assets/images/google-meet.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?><svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 122.88 101.11" style="enable-background:new 0 0 122.88 101.11" xml:space="preserve"><style type="text/css"><![CDATA[

```

Comment:
This was an svg that I found online. It's not from the brand kit that was posted in the issue, because that kit only gives png files. That makes our code nastier, so I went for the same exact image but in svg form.